src/js/components/put-away/SplitLineModal.jsx

```diff
@@ -144,7 +144,7 @@ class SplitLineModal extends Component {
    */
   isBinSelected() {
     return _.every(this.state.splitItems, splitItem =>
-      splitItem.putawayLocation.id);
```

Comment:
I think you can use _.get('splitItem.putawayLocation.id') instead, because there can be a situation when there is a putawayLocation with empty id